go.mod

```diff
@@ -1,3 +1,11 @@
 module github.com/TheForgotten69/goinsta/v2
 
 go 1.15
+
+require (
+	github.com/stretchr/testify v1.7.0
+	github.com/tcnksm/go-input v0.0.0-20180404061846-548a7d7a8ee8
```

Comment:
Isn't it only used on examples ?

Reply:
Seems so. But that is a good practice to keep go.mod up to sync with all dependencies. In other case examples should be moved to separate module.